.gitpod.yml

```
@@ -26,19 +22,22 @@ vscode:
     - stylelint.vscode-stylelint
     - dbaeumer.vscode-eslint
     - evgeniypeshkov.syntax-highlighter
-    - redhat.vscode-yaml
     - ms-azuretools.vscode-docker
     - Catppuccin.catppuccin-vsc
     - PKief.material-icon-theme
     - oderwat.indent-rainbow
     - formulahendry.auto-rename-tag
-    - eamodio.gitlens
+    - swellaby.vscode-rust-test-adapter
+    - belfz.search-crates-io
+    - hbenl.test-adapter-converter
+    - hbenl.vscode-test-explorer
+
```

```suggestion
    - hbenl.vscode-test-explorer
    - eamodio.gitlens
```
Also, I think the gitlens extension would really add up for a good developer experience, as this makes doing and viewing git changes easier. What do you think ?